inc/plugin.class.php

```diff
@@ -51,6 +51,12 @@ class Plugin extends CommonDBTM {
 
    static $rightname = 'config';
 
+   /**
+    * Loaded plugin list, indexed by their ID.
+    *
+    * @var null|string[]
+    */
+   private static $loaded_plugins;
```

Comment:
As we reload the plugin list on each page, it does not make any sense to cache it in the cache storage.
1) This makes uselss access to the cache storage for a small string array that will be regenerated realy often.
2) This makes storing this on a shared space that have to handle concurrent accesses.